.github/workflows/arviz_compat.yml

```diff
@@ -38,8 +38,15 @@ jobs:
           channel-priority: strict
           environment-file: environment-dev.yml
           use-only-tar-bz2: true # IMPORTANT: This needs to be set for caching to work properly!
-      - run: |
+      - name: Install pymc3
+        run: |
           conda activate testenv
-          conda remove arviz -y
+          pip install -e .
```

Comment:
it's probably good to check this works, in case there are any errors in setup.py - it only takes about 5 seconds during CI anyway